### DISCUSSION TITLE
Compilation error while zen EvaluateExpression

### DISCUSSION THREAD

**Discussion by shravani154** [gorules/zen-go#20] [Q&A]
Hello,

I'm new to zen-go and I'm attempting to evaluate an simple expression using it, but I'm encountering an error. Please let me know if there's anything I might be overlooking.

````
package main
import (
	"fmt"
	"github.com/gorules/zen-go"
)

func main() {
	r1, err := zen.EvaluateExpression[int]("1 + 1", nil)
	if err != nil {
		fmt.Println("Error evaluating expression:", err)
		return
	}
	fmt.Println("Result:", r1)
}



ERROR:
.\main.go:9:17: undefined: zen.EvaluateExpression

**Comment by divyeshk31** [comment]
must be some error with dependency. working fine for me. have you used go mod init to download the dependencies correctly?
<img width="543" alt="Screenshot 2024-07-01 at 7 17 03 PM" src="https://github.com/gorules/zen-go/assets/14954959/e36ecd1e-ab48-4f20-b172-75b0260af3ac">

**Reply by shravani154** [reply]
Yes I did. The issue is only with windows. The same project when tried to run on mac didn't throw any error.
